test/static/vast_ad_buffet_with_error.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<VAST version="3.0">
+    <Ad id="5566276">
+        <InLine>
+            <AdSystem>ExoClick</AdSystem>
+            <AdTitle/>
+            <Impression id="exotr"><![CDATA[https://syndication.realsrv.com/vregister.php?a=vimp&tracking_event=impression&idzone=4890178&2f619b13242072d9159f66efe8cd218d=tsVuZ8uHLnt4ctvDtq48PXLz64ctdlTlK8E.fjzu48em7r33ceHjW1NZLXThmYchrgbjYlesYecz6cddUFbi781VcrEjmbVXDhuampn3OXzNRwNRbmppNcDbDdrlNcFTlOfbt24dOuuBuexmOCp9ynPpx4.OXXXA3VBW5n25c.HLvrgbxmlcz6ePPDj38a4G2mK3HpqcM.vDxrgbaYknYgelz69O3Dpz4a4G7WKYGK4Jpc._nj27.eHDXA3NVnx664G2aZrqnKc.WuBtty2BpzPhrgbaYppgcpz4a4G4Kp8.PLzy11WM58NdrEdjmfDdw49OmuexmOCp9ylelitzPvw1z2MxwVPuUrtWU0uStYZgona2mJJ2IHpV2rKaXJWsM0TwNbl7T7Erzi9cy89jMcFT7lOfDd54.eWty9p9iV5xeuZeVyu6amLPjrYbXrwncz4.Nbs1MjFeeuBuVyu6amLPjramslrpwXmpgeglYjzMOQ1v11zr3ruzU3MUtuNruzU564G56Zm7Gq12mK3HpqcM.XDXPTA1BK8vJM25Hn01v11z1Z8ddTVLjkq9LlU0dlcE0ueuypyleBvPtz4OceHTn06.OnTtu49fPThz6eOPDv25cfHbz512UxrvsVP5t9OLfhpvz4d4cGeXDo558cOHnz4cad8dOXdzXBJPS5VVBNKvVWxXZVnx1wST0uVVQTSrwS2sRwNr0uMVTS58NdLjrlLlK9UFbi781VcrEjmethtmOZqLPhrgbmddcpz4a4G42JW4JXl52HnM.Gty9xqyuCaVeuCRzPhu4cdcDbbFbDTktblOfLXA20xTTA5SvVNZS05nw1yzVNUwT158NcErUz0sFcy8kzbmfDXW5VWvJM25nw10uPQTSrvOTSsSOLwN58u_Pr56cdc9M1.C9VbFdlWe3jrgbnYprlcpz4a2oK8F3nJpWJHF4G8.Xfn189OeuVythqyCvBeema_BevCdzPXK5Ww1ZBXgvPTNfgu25U1TBPXBNLnrYbZjmaiXtcpz1wST0uVVQTSrsRxrwS2sRwNr0uMVTS1Z8NdVjPLPhrqsZ558NdTVME9a9eE7meupqmCeteViRzPXU1TBPWva5TnrZpmuqcpXtcpz4a7ac.GuCWtymViPPhrlmXdslbqz4a4G6XKp5paoLXF42MJrK8.GuBuSyOuDGaVzPhrsqcpXaYnnglez467KnKV2mJ54JXl3aXKLHJWsM.PTpw1uSMQRrwVT58NdTVME9a7blbEEefbXU1TBPWva5TVBNLnx1s2Ux567bLIG8.Pbv17ee_Ppy48eXnv54cO_Htzc48WmXOzLjDOuuCRyqtiSfPj279e3nvz6ctbU00UDjU0tTktefGA-]]></Impression>
```

Comment:
this shouldn't have the "https://syndication.realsrv.com/vregister.php?". should be something dummy here